server/routes/groups.js

```diff
@@ -54,17 +55,13 @@ router.post('/api/add-group', (req, res, next) => {
     .save()
     .then((review) => {
       console.log("review", review.id);
-      // res.send(review.id);
       Response.forge({
         review_id: review.id,
       })
       .save()
+      res.status(200).json(review)
     })
   })
-  .then(() => {
-    res.sendStatus(200);
-    // res.status(200).json(group) //these seemed to be causing errors..? it might have been something else though
```

Comment:
Responding to the comment you made inline here. I think the reason you were seeing errors is because the `group` object (just the response data) was undefined since it is out of scope of the `.then` function where it originally lived. By moving the `res.status(200).json(review)` back into a scope with a defined response param, we have access to send that response object along with a status code. Now it is "review".